src/util/valuetransformer.h

```diff
@@ -18,9 +19,7 @@ class TransformAdd : public TransformNode {
   public:
     TransformAdd(double addend) : m_addend(addend) {}
 
-    double transform(double argument) const {
-        return argument + m_addend;
-    }
+    double transform(double argument) const { return argument + m_addend; }
```

Comment:
Thew was nothing wrong with the old coding style. 
I prefer a one statement per line. 
We use it all over in Mixxx and it offers the best readability.